homework_10,11.py

```diff
@@ -0,0 +1,67 @@
+# 1. Написати власний декоратор, задачею якого має бути друк назви функції і часу, коли вона була викликана.
+from datetime import datetime
+
+now = datetime.now()
+current_time = now.strftime("%H:%M:%S")
+
+def my_decorator(func):
+    def deco_func(*args):
+        now = datetime.now()
+        current_time = now.strftime("%H:%M:%S")
+        print(f"Function name: {func}")
+        print(f"Function call time: {current_time}")
+        func(*args)
```

Comment:
Use `return` statement here to be able to decorate functions that return some value. And add **kwargs here